conf/defaults.ini

```diff
@@ -550,6 +550,10 @@ sampler_type = const
 # and indicates the initial sampling rate before the actual one
 # is received from the mothership
 sampler_param = 1
+# Whether or not to use Zipkin span propagation (x-b3- HTTP headers).
+zipkin_propagation = false
+# If zipkin propagation is enabled, whether or not to use shared RPC spans.
+shared_zipkin_spans = true
```

Comment:
Please change the default value for the setting to `false`. 

The reason we want all bool settings to be false is because `false` is the default value for bools. https://dave.cheney.net/2013/01/19/what-is-the-zero-value-and-why-is-it-useful

Sorry that I forgot that in my last review. Looks good otherwise! I'll merge this once this is fixed.

Reply:
Done.  We treat unset as false in the Go code now as well.

Reply:
I still think passing the span could be the default behavior. So instead of having a setting for enabling it, there could be a setting for disabling it. As such, the config value can default to false and behavior can still be enabled. 

Ex https://github.com/golang/go/blob/master/src/crypto/tls/common.go#L523

Sorry if that was unclear.

Reply:
Smart!  I have made that adjustment.  Feel free to critique the documentation or parameter naming.

Reply:
Seems like it's still disabled by default. If we swap the meaning of the settings it should probably be renamed `disable_shared_zipkin_spans`

Reply:
Any chance you could update the PR? Otherwise, I'll fix it once I have spare time.